crypto/src/asset/registry.rs

```diff
@@ -293,5 +293,14 @@ pub static REGISTRY: Lazy<Registry> = Lazy::new(|| {
                 denom::Inner::new(format!("lpnft_{}", data), vec![])
             }) as for<'r> fn(&'r str) -> _,
         )
+        .add_asset(
+            // Note: this regex must be in sync with ProposalNft::try_from
+            "^proposal_(?P<data>(?P<proposal_id>[0-9]+)_(?P<proposal_state>voting|withdrawn|passed|failed|vetoed))$",
+            &[ /* no display units - nft, unit 1 */ ],
+            (|data: &str| {
+                assert!(!data.is_empty());
+                denom::Inner::new(format!("proposal_{}", data), vec![])
+            }) as for<'r> fn(&'r str) -> _,
+        )
```

Comment:
Adding five new families of assets, the regex: `proposal_([0-9]+)_(voting|withdrawn|passed|failed|vetoed)` representing the lifecycle of a proposal from the point of view of its proposer.